sdk/schemaregistry/schema-registry-avro/package.json

```diff
@@ -91,7 +91,7 @@
     "@types/chai-as-promised": "^7.1.0",
     "@types/lru-cache": "^7.4.0",
     "@types/mocha": "^7.0.2",
-    "@types/node": "^12.0.0",
```

Comment:
What a nice feature! Just want to make sure that we don't require our consumer to move to v16, right?

Reply:
`cause` is available starting in node from v16.9 so customers of earlier node versions will not have access to `cause` but their code will function fine otherwise as evident in the ci pipeline for node v12 in this PR.